### DISCUSSION TITLE
Exposed IP - Exposure management

### DISCUSSION THREAD

**Discussion by EndPoint-dev-Brazil** [CrowdStrike/falconpy#1197] [Q&A] [answered]
I would like some help to notify assets with IP exposed to the internet extracted from the menu in Exposure Management, there is a filter called "internet exposure" shows assets with IP exposed to the internet, is there any way to create a notification about this filter.

**Comment by jshcodes** [comment, marked as answer]
Hi @EndPoint-dev-Brazil -

Thank you for the question!

The documentation for this filter was just released as part of the [`1.4.4`](https://github.com/CrowdStrike/falconpy/releases/tag/v1.4.4) updates: https://falconpy.io/Service-Collections/Exposure-Management.html#available-filters-exact-match

You could also accomplish this by using a Fusion workflow.
![image](https://github.com/user-attachments/assets/7c1fe1b5-cfd1-49ac-937a-b160d804e745)
